tctl.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: tctl
   version: 1.18.0
-  epoch: 9
+  epoch: 1
```

Comment:
epoch cannot go backwards, e.g. 9 to 1

Reply:
yeah, but somebody merged the PR with epoch 9 by mistake before 9 could be reverted back to 1, should we leave it at 9 now?

Reply:
Yeah, I think it went unnoticed here, I think reviewers had left thought but somehow it got merged. https://github.com/wolfi-dev/os/pull/8897/files#r1410816853

Reply:
this is not needed anymore lets close this